main.go

```diff
@@ -80,7 +76,7 @@ func (conn *Connection) Connect() {
 	logger.Trace("Connected")
 	conn.shutdown = make(chan bool)
 	conn.shutdownFinal = make(chan bool)
-	
+
 	conn.target = stringToNetId(conn.netid)
 	conn.target.port = 801
```

Comment:
Could it be that we need to update this to `port` as well?

Reply:
You are right. I replaced that as well with a new commit.
I tested it on a separate system and forgot to take both changes into the repo on my development machine.